docs/components/badges.md

```diff
@@ -10,10 +10,12 @@ section: css
 
 
 # Badges
+<p class="lead">
+  A badge is generally used to categorize content.
+</p>
 
-Badge uses `display: inline-block` by default.
-
-Usage example:
+## Usage
+The badge component has `display: inline-block` by default.
```

Comment:
O que tu acha de algo como "To the badge component a `display: inline-block` rule is applied by default"?